src/components/InteractiveSearch.js

```diff
@@ -9,6 +9,14 @@ import Typography from 'components/Typography'
 import Lottie from 'components/Lottie'
 import { useInView } from 'react-intersection-observer'
 
+const NEXT_PUBLIC_DEMO_MOVIES_HOST = process.env.NEXT_PUBLIC_DEMO_MOVIES_HOST
+  ? process.env.NEXT_PUBLIC_DEMO_MOVIES_HOST
+  : 'http://0.0.0.0:7700'
```

Comment:
You can make it more readable by doing so:

```
const NEXT_PUBLIC_DEMO_MOVIES_HOST = process.env.NEXT_PUBLIC_DEMO_MOVIES_HOST ||
'http://0.0.0.0:7700'
```

Reply:
Done.